2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2024 @@
+live up to the standards and remain a central component of the research
+landscape.
+
+University computing centers and supercomputing facilities need to
+receive earmarked resources for the support of sustainable software
+development. This funding is necessary to provide continuous integration
+services, a hardware portfolio for development, testing and benchmarking
+software, as well as personnel for training domain researchers in
+software design and the proper usage of the services.
+
+The creation and maintenance of training material for general research
+software engineering education and the software-specific documentation
+and tutorial creation needs to be reflected in funding opportunities.
+This can either happen by dedicating modules of research or software
+grants to providing support and the generation of training material, or
+by opening funding schemes focusing on interdisciplinary software
+development education. The latter may include research that looks at
+research software development as a process to analyze which measures,
+interactions, and team compositions make research software successful.
+Additionally, funding instruments fostering the formation of research
+software communities have to be established.
+
+Which Infrastructure is Needed to Sustain Research Software?
+============================================================
+
+> As the *hal9k* community grows, so does the need for infrastructure.
+> Kim and her team collaborate with the National Research Software
+> Engineering Consortium to set up *hal9k* on the Consortium’s
+> distributed TigHub instance, and organize world-wide access to it via
+> the NRSEC-AAI. Going forward, the Consortium’s Research Software Hub -
+> a registry and Software Heritage Archive-based long-term repository
+> for research software on a national level - ingests *hal9k* releases
+> with complete metadata: citation metadata, the *hal9k* provenance
+> graph and computational environment information, ORCID iDs, etc. and
+> provides its own DOIs for versions under a concept DOI. The community
+> reviews all code and documentation changes that are contributed to
+> *hal9k* via the central TigHub, and the Hub’s CI system Alfred builds,
+> tests, and pushes new releases automatically to the registered
+> supercomputing clusters. Especially the community efforts become
+> better and more streamlined by the day, as research software
+> development training is now offered as part of most curricula, and
+> skilled RSEs are now much easier to find and hire by research
+> institutions.
+
+Project Management Tools
+------------------------
+
+Research software is developed by individual researchers, in small teams
+within a single institution, or in larger teams distributed across
+multiple institutions. In particular if software development is
+distributed across institutions, there exists an urgent need for
+frameworks and tools enabling col
```

Comment:
Felix: please address @StephanJanosch' comment above regarding telemetry to track usage here.

Reply:
Transferred to Overleaf.